ecs-logging-core/src/test/java/co/elastic/logging/AbstractEcsLoggingTest.java

```diff
@@ -124,8 +142,14 @@ public boolean putNdc(String message) {
         return false;
     }
 
+    public ParameterizedLogSupport getParameterizedLogSettings() {
+        return ParameterizedLogSupport.BRACKETS_ONLY;
+    }
+
     public abstract void debug(String message);
 
+    public abstract void debug(String message, Object[] logParams);
```

Comment:
Just an idea: may be a vararg so that callers can do `debug("{} is not {}", 1, 2)` vs `debug("{} is not {}", new Object[]{1, 2})`
```suggestion
    public abstract void debug(String message, Object... logParams);
```

Reply:
Yes I agree, I've refactored it to varargs. 

What do you think of the place of the testSimpleParameterizedLog method? I was not sure about whether I should place the test in the abstract base class AbstractEcsLoggingTest (because testSimpleLog is also already there), or make a test in each formatter project.

Reply:
I really like the abstract way of handling this! IMO, the test classes in each formatted project should be as simple as possible and ideally only extend the abstract test and implement its abstract methods.

Reply:
Ok nice! :)